noobaa_sa/account.py

```diff
@@ -62,18 +67,47 @@ class NSFSAccount(Account):
 
     def create(
         self,
-        account_name,
-        access_key,
-        secret_key,
+        account_name="",
+        access_key="",
+        secret_key="",
+        email="",
+        allow_bucket_creation=True,
+        uid=0,
+        gid=0,
         config_root=None,
         fs_backend=constants.DEFAULT_FS_BACKEND,
     ):
         """
         Account creation using file
 
         Args:
-            config_root (str): Path to config root
-        """
+            account_name (str): name of the account
+            access_key (str): access key for the account
+            secret_key (str): secret key for the account
+            email (str): email for the account
+            allow_bucket_creation (bool): allow bucket creation
+            uid (int): user ID
+            gid (int): group ID
+            config_root (str): path to config root
+            fs_backend (str): filesystem backend
+
+        Returns:
+            tuple:
+                account_name (str): name of the account
+                access_key (str): access key for the account
+                secret_key (str): secret key for the account
+
+        """
+
+        # Set default values if not provided
+        if not account_name:
+            account_name = generate_unique_resource_name(prefix="account")
+        if not access_key:
+            access_key = generate_random_hex()
+        if not secret_key:
+            secret_key = generate_random_hex()
+        if not email:
+            account_email = config.ENV_DATA["email"]
```

Comment:
As per the latest build, Access_key should contains Uppercase and lower case characters.
Please refer https://github.com/red-hat-storage/noobaa-sa-ci/pull/19

Reply:
resolved in https://github.com/red-hat-storage/noobaa-sa-ci/pull/16/commits/de02bcff175fb2a11114113c0c5786da75a7f636